### DISCUSSION TITLE
iOS and Android show "0.1" for version rather than actual release version

### DISCUSSION THREAD

**Discussion by itsskyballs** [ppy/osu#21498] [Q&A] [answered]
The Latest Version has a Incorrect Version if you go to app info or check the app package i think the version was not editied or forgotten

Not Correct Version:0.1
Version:Any version or later
![Screenshot_20221201-205926-1-1-1](https://user-images.githubusercontent.com/111575685/205285792-d3483fb1-1218-496d-8893-0e8b5fb98c89.jpg)

**Comment by yesseruser** [comment]
That happens to me as well. Maybe it's just because it's not downloaded through the app store. Anyways, this is not such an important bug since you can still view the version in-game.

**Reply by itsskyballs** [reply]
still 0.1 because after installed the game and go to app info it will display the incorrect version
without editing the version unlike the .net framework(i mean no not .net fw)

**Reply by bdach** [reply]
@stevenkiethanete cannot reproduce

![image](https://user-images.githubusercontent.com/20418176/210082471-b2b81a03-f2c0-4422-a022-a697e16e4424.png)
![image](https://user-images.githubusercontent.com/20418176/210082491-3e8b307d-30ac-42dd-aca5-1f7930a99b37.png)
![image](https://user-images.githubusercontent.com/20418176/210082531-d7424160-ebc2-49ca-9be7-249e4930eaa6.png)
![image](https://user-images.githubusercontent.com/20418176/210082554-c86b196b-c44b-4c40-953a-7e21c79d4773.png)

**Reply by itsskyballs** [reply]
yea
dont know why

**Comment by bdach** [comment, marked as answer]
Android should be fixed via https://github.com/ppy/osu/pull/21878 (iOS already displays correct version).

**Reply by itsskyballs** [reply]
mine not working
because it will report 0.1
but yes #21878

**Comment by itsskyballs** [comment]
i tried to modify the version of the game and still report '0.1' version

**Comment by zzahkaboom24** [comment]
![Screenshot_2023-01-08-03-20-51-669_com miui securitycenter](https://user-images.githubusercontent.com/42496614/211177786-13a61ac2-5679-4afb-aa0f-c839dda6ffdb.jpg)

Cannot reproduce as well. Versioning matches what's ingame

**Reply by itsskyballs** [reply]
i showed it on my mom's phone and the version is  correct
and why is not answered

**Reply by frenzibyte** [reply]
Because iOS seems to have not been updated yet.

**Comment by yesseruser** [comment]
The versioning must have been changed in an update.

**Reply by frenzibyte** [reply]
No idea what this comment implies. Do you mean it's broken again? Which platform, is it iOS or Android? What's the version that's displayed now?

**Reply by yesseruser** [reply]
No, the version is correct now. On Android.

**Reply by itsskyballs** [reply]
> No idea what this comment implies. Do you mean it's broken again? Which platform, is it iOS or Android? What's the version that's displayed now?

Its updated now
